<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>999</width>
    <height>530</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>11</number>
    </property>
    <property name="topMargin">
     <number>11</number>
    </property>
    <property name="rightMargin">
     <number>11</number>
    </property>
    <property name="bottomMargin">
     <number>11</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QFrame" name="frame">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>176</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="dashboardButton">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>40</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Poppins Medium</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Dashboard</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="skywardButton">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>40</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Poppins Medium</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Skyward</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="gpaButton">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>55</width>
          <height>40</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Poppins Medium</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>GPA</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>MS UI Gothic</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>îœ“</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QStackedWidget" name="stackedWidget_2">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="dashboardPage">
       <layout class="QGridLayout" name="gridLayout_4">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_5"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="skywardPage">
       <layout class="QGridLayout" name="gridLayout_3">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>5</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QListWidget" name="classesFilter">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <family>Poppins</family>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="frameShape">
               <enum>QFrame::NoFrame</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Plain</enum>
              </property>
              <property name="lineWidth">
               <number>1</number>
              </property>
              <property name="editTriggers">
               <set>QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed</set>
              </property>
              <property name="tabKeyNavigation">
               <bool>true</bool>
              </property>
              <property name="dragEnabled">
               <bool>false</bool>
              </property>
              <property name="alternatingRowColors">
               <bool>false</bool>
              </property>
              <property name="currentRow">
               <number>-1</number>
              </property>
              <item>
               <property name="text">
                <string>All</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>English</string>
               </property>
               <property name="font">
                <font>
                 <weight>50</weight>
                 <bold>false</bold>
                </font>
               </property>
               <property name="flags">
                <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Environmental Science</string>
               </property>
               <property name="flags">
                <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Pre Calc</string>
               </property>
               <property name="flags">
                <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
               </property>
              </item>
              <item>
               <property name="text">
                <string>US History</string>
               </property>
               <property name="flags">
                <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Game Design</string>
               </property>
               <property name="flags">
                <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Computer Science</string>
               </property>
               <property name="flags">
                <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QStackedWidget" name="stackedWidget">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="currentIndex">
               <number>1</number>
              </property>
              <widget class="QWidget" name="page">
               <layout class="QGridLayout" name="gridLayout">
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item row="0" column="0">
                 <widget class="QTreeWidget" name="classViewWidget">
                  <property name="font">
                   <font>
                    <family>Poppins Medium</family>
                   </font>
                  </property>
                  <property name="alternatingRowColors">
                   <bool>true</bool>
                  </property>
                  <property name="rootIsDecorated">
                   <bool>true</bool>
                  </property>
                  <property name="allColumnsShowFocus">
                   <bool>false</bool>
                  </property>
                  <property name="headerHidden">
                   <bool>false</bool>
                  </property>
                  <attribute name="headerCascadingSectionResizes">
                   <bool>false</bool>
                  </attribute>
                  <column>
                   <property name="text">
                    <string>Name</string>
                   </property>
                   <property name="font">
                    <font>
                     <weight>75</weight>
                     <bold>true</bold>
                    </font>
                   </property>
                  </column>
                  <column>
                   <property name="text">
                    <string>Grade</string>
                   </property>
                   <property name="font">
                    <font>
                     <weight>75</weight>
                     <bold>true</bold>
                    </font>
                   </property>
                  </column>
                  <column>
                   <property name="text">
                    <string>Due</string>
                   </property>
                   <property name="font">
                    <font>
                     <weight>75</weight>
                     <bold>true</bold>
                    </font>
                   </property>
                  </column>
                  <item>
                   <property name="text">
                    <string>1.1-1.3 Quiz</string>
                   </property>
                   <property name="text">
                    <string>95</string>
                   </property>
                   <property name="text">
                    <string>09/04/2022</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Essay</string>
                   </property>
                   <property name="text">
                    <string>97</string>
                   </property>
                   <property name="text">
                    <string>09/09/2022</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Review</string>
                   </property>
                   <property name="text">
                    <string>100</string>
                   </property>
                   <property name="text">
                    <string>09/01/2022</string>
                   </property>
                  </item>
                 </widget>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="page_2">
               <layout class="QGridLayout" name="gridLayout_2">
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item row="0" column="0">
                 <widget class="QTableWidget" name="skywardTable">
                  <property name="font">
                   <font>
                    <family>Poppins Medium</family>
                   </font>
                  </property>
                  <property name="frameShape">
                   <enum>QFrame::NoFrame</enum>
                  </property>
                  <property name="frameShadow">
                   <enum>QFrame::Sunken</enum>
                  </property>
                  <property name="editTriggers">
                   <set>QAbstractItemView::NoEditTriggers</set>
                  </property>
                  <property name="tabKeyNavigation">
                   <bool>false</bool>
                  </property>
                  <property name="alternatingRowColors">
                   <bool>true</bool>
                  </property>
                  <property name="textElideMode">
                   <enum>Qt::ElideRight</enum>
                  </property>
                  <property name="verticalScrollMode">
                   <enum>QAbstractItemView::ScrollPerPixel</enum>
                  </property>
                  <property name="horizontalScrollMode">
                   <enum>QAbstractItemView::ScrollPerPixel</enum>
                  </property>
                  <property name="gridStyle">
                   <enum>Qt::SolidLine</enum>
                  </property>
                  <property name="sortingEnabled">
                   <bool>false</bool>
                  </property>
                  <property name="cornerButtonEnabled">
                   <bool>true</bool>
                  </property>
                  <attribute name="horizontalHeaderDefaultSectionSize">
                   <number>56</number>
                  </attribute>
                  <row>
                   <property name="text">
                    <string>English</string>
                   </property>
                  </row>
                  <row>
                   <property name="text">
                    <string>Environmental Science</string>
                   </property>
                  </row>
                  <row>
                   <property name="text">
                    <string>Pre Calc</string>
                   </property>
                  </row>
                  <row>
                   <property name="text">
                    <string>US History</string>
                   </property>
                  </row>
                  <row>
                   <property name="text">
                    <string>Game Design</string>
                   </property>
                  </row>
                  <row>
                   <property name="text">
                    <string>Computer Science</string>
                   </property>
                  </row>
                  <column>
                   <property name="text">
                    <string>1st</string>
                   </property>
                  </column>
                  <column>
                   <property name="text">
                    <string>2nd</string>
                   </property>
                  </column>
                  <column>
                   <property name="text">
                    <string>3rd</string>
                   </property>
                  </column>
                  <column>
                   <property name="text">
                    <string>SEM1</string>
                   </property>
                  </column>
                  <column>
                   <property name="text">
                    <string>4th</string>
                   </property>
                  </column>
                  <column>
                   <property name="text">
                    <string>5th</string>
                   </property>
                  </column>
                  <column>
                   <property name="text">
                    <string>6th</string>
                   </property>
                  </column>
                  <column>
                   <property name="text">
                    <string>SEM2</string>
                   </property>
                  </column>
                  <column>
                   <property name="text">
                    <string>Date</string>
                   </property>
                  </column>
                 </widget>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
            <item>
             <widget class="QFrame" name="weeksFilterFrame">
              <property name="autoFillBackground">
               <bool>true</bool>
              </property>
              <layout class="QHBoxLayout" name="weeksFilterFrame_2">
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QListWidget" name="weeksFilter">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>37</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <family>Poppins Medium</family>
                  </font>
                 </property>
                 <property name="frameShape">
                  <enum>QFrame::NoFrame</enum>
                 </property>
                 <property name="verticalScrollBarPolicy">
                  <enum>Qt::ScrollBarAlwaysOff</enum>
                 </property>
                 <property name="horizontalScrollBarPolicy">
                  <enum>Qt::ScrollBarAlwaysOff</enum>
                 </property>
                 <property name="flow">
                  <enum>QListView::LeftToRight</enum>
                 </property>
                 <property name="resizeMode">
                  <enum>QListView::Adjust</enum>
                 </property>
                 <property name="viewMode">
                  <enum>QListView::ListMode</enum>
                 </property>
                 <property name="currentRow">
                  <number>0</number>
                 </property>
                 <item>
                  <property name="text">
                   <string>All</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>1st</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>2nd</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>3rd</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>4th</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>5th</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>6th</string>
                  </property>
                 </item>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="gpaPage">
       <layout class="QGridLayout" name="gridLayout_5">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_6"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="refreshButton">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>MS UI Gothic</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>îœ¬</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lastRefreshedLabel">
        <property name="font">
         <font>
          <family>Poppins Medium</family>
          <pointsize>7</pointsize>
          <italic>false</italic>
         </font>
        </property>
        <property name="text">
         <string>Last Refreshed: 09/07/2022  09:54 am</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>999</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
